index.html

```diff
@@ -152,8 +164,7 @@ <h5 class="card-title">Basic Python in 8 Hours </h5>
           <img src="images/firbease.png" class="card-img-top" alt="...">
           <div class="card-body">
             <h5 class="card-title">How to Use Firebase?</h5>
-            <p class="card-text">Learn Firebase using Flutter Dart and JavaScript. Performing all the CRUD operations
-              with Real and Cloud Storage.</p>
+            <p class="card-text">Learn Firebase using Flutter Dart and JavaScript.</p>

```

Comment:
change the text 

`Learn Firebase using Flutter Dart and JavaScript. Performing all the CRUD operations with Real and Cloud Storage`